### PR TITLE
Add description to invert color

### PR DESCRIPTION
**Pull request by guilhermefelipecgs**
Also change "Mipmaps" from h2 to h3.

### CODE REVIEW

File: getting_started/workflow/assets/importing_images.rst [hunk 1]
```diff
@@ -149,6 +149,9 @@ HDR as SRGB
 
 Some few HDR files are broken and contain SRGB color data. It is advised to not use them but, in the worst case, toggling this option on will make them look right.
 
+Invert Color
+~~~~~~~~~~~~
+When this option is enabled, the colors of the image will be reversed. This option is useful for example to convert a height map generated by external programs to depth map to use with :ref:`doc_spatial_material`.
```
Comment (NathanLovato) [inline getting_started/workflow/assets/importing_images.rst]: Add a blank line between titles and paragraphs, and please avoid using the passive voice: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html

Comment (NathanLovato) [inline getting_started/workflow/assets/importing_images.rst]: Add a blank line between titles and paragraphs, and please avoid using the passive voice: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html

Comment (guilhermefelipecgs) [reply getting_started/workflow/assets/importing_images.rst]: >  avoid using the passive voice

This way?
```
When enabled, the colors of the image will be reversed. This is useful for example to convert a height map generated by external programs to depth map to use with :ref:`doc_spatial_material`.
```

Comment (NathanLovato) [reply getting_started/workflow/assets/importing_images.rst]: You can also remove the first one: `Reverses the image's color`

Comment (guilhermefelipecgs) [reply getting_started/workflow/assets/importing_images.rst]: Ok, thanks :+1:

**Comment by NathanLovato** [comment]
Thanks :)

**Pull request merged**